medicai/conversation/routes.py

```diff
@@ -0,0 +1,35 @@
+from flask import request, jsonify
+from flask_jwt_extended import create_access_token, jwt_required, get_jwt_identity
+from flask_cors import CORS, cross_origin
+from medicai.models.conversation import Conversation
+from medicai.models.user import User
+from medicai.extensions import db
+from medicai.conversation import bp
+from datetime import datetime
+
+@bp.route('/getLatest/<int:userId>', methods=['GET'])
+@cross_origin()
+def get_latestConversation(userId):
+    current_user = User.query.filter_by(id=userId).first()
```

Comment:
`user = User.query.get(id)` is better than doing filter and get first.

Reply:
fixed